### ISSUE TITLE
Warnings from light.exe on msi task

### ISSUE THREAD

**Issue by azhawkes** [i-net-software/SetupBuilder#24]
Thanks for this awesome plugin -- it has already made my life a lot easier!

I'm getting the following warnings from light.exe and wanted to confirm whether they're intentional:

```
warning LGHT1079 : The cabinet 'media1.cab' does not contain any files.  If this installation contains no files, this warning can likely be safely ignored.  Otherwise, please add files to the cabinet or remove it.
warning LGHT1076 : ICE61: This product should remove only older versions of itself. No Maximum version was detected for the current product. (WIX_UPGRADE_DETECTED)
```

LGHT1079 looks fairly harmless and I think it's just because WsxFileBuilder.java is adding a cabinet no matter what. Is this cabinet ever non-empty? I don't see any files being added to it.

LGHT1076 is a bit more worrisome. I see the following in WsxFileBuilder:

```java
        // MajorUpgrade
        if( task.getMultiInstanceCount() <= 1 ) { 
            Element update = getOrCreateChild( product, "MajorUpgrade" );
            addAttributeIfNotExists( update, "AllowDowngrades", "yes" );
        }
```

What if I just want a single instance (multiInstanceCount == 1), but to not allow downgrades? Based on stuff I am reading (and this warning from light.exe), I'd be more comfortable if the MajorUpgrade element were like this:

```xml
    <MajorUpgrade AllowDowngrades="no"/>
```

I know very little about Windows Installer and Wix, so it's possible I'm missing something. Was this intentional or would you accept a PR to make AllowDowngrades a configurable setting on the "msi" task?

**Comment by Horcrux7** [comment]
**LGHT1079:** This is an effect of an performance hack. The SetupBuilder supports multiple languages. This will create through a diff file to the main language. To improve the speed it create an empty msi file for the different languages. Create the diff file. Create the final msi file with files and add the diff files. I think we should hide this warning.

**LGHT1076:** This is philosophic question. Why should I block the user to install an older version if the current version is not working for the user? The user can also uninstall and install the new version. An update is equals to uninstall the old installation and install the current version. But if you like this then you can do it with an [wsx template file](https://github.com/i-net-software/SetupBuilder/wiki/Create-a-MSI-with-WSX-template). If you set AllowDowngrades="no" then you must also set an error message in the wsx template that the user will see if he try to downgrade.

**Comment by azhawkes** [comment]
I did some more testing on my end around downgrades, and they did not seem to have the bad side effects that certain Wix forum posts implied they did. I agree with your philosophical point. It's interesting that the Wix team thought that allowing downgrades deserved issuing a *warning*. After my testing this morning I've decided that downgrades are probably harmless so I'm going to ignore that warning.

Thanks for your reply!
